gettsim/interface.py

```diff
@@ -94,6 +98,9 @@ def compute_taxes_and_transfers(
     }
     _fail_if_targets_not_in_functions(functions, targets)
 
+    if rounding:
```

Comment:
We should add a comment that it is very important to handle the rounding before the partialling. Just to prevent bugs in future refactoring.